lib/fuzzy-provider.coffee

```diff
@@ -1,8 +1,7 @@
 _ = require 'underscore-plus'
 Suggestion = require './suggestion'
 fuzzaldrin = require 'fuzzaldrin'
-{CompositeDisposable} = require 'event-kit'
-{TextEditor}  = require 'atom'
+{TextEditor, CompositeDisposable}  = require 'atom'
```

Comment:
do we really want to rely on the composite disposable exposed from atom?  I'm not sure of a good argument for doing this.